packages/perseus/src/widgets/radio/base-radio.jsx

```diff
@@ -247,7 +247,9 @@ function BaseRadio(props: Props): React.Node {
         <fieldset
             className={`perseus-widget-radio-fieldset ${responsiveClassName}`}
         >
-            <legend className="perseus-sr-only">{instructions}</legend>
+            {shouldShowInstructions && (
```

Comment:
Gated the legend element based on shouldShowInstructions so that the screen reader experience matches the sighted experience. When satStyling is true, visible instructions were hidden. This addition makes it so the screen reader instructions are also hidden when satStyling is true.